gcfit/probabilities/probabilities.py

```diff
@@ -496,11 +484,15 @@ def likelihood_pulsar_orbital(model, pulsars, cluster_μ, coords, use_DM=False,
         # Interpolate the likelihood value from the overall distribution
         # ------------------------------------------------------------------
 
-        prob_dist = interp.interp1d(
-            (lin_domain / Pb) + PdotP_pm + PdotP_gal, conv,
-            assume_sorted=True, bounds_error=False, fill_value=0.0
+        PdotP_tot = ((lin_domain / Pb) + PdotP_pm + PdotP_gal).to_value(1 / u.s)
+
+        # normalize conv to the PdotP_tot domain
+        conv /= util.QuantitySpline(PdotP_tot, conv, k=1, s=0, ext=1).integral(
+            -np.inf, np.inf
         )
 
+        prob_dist = util.QuantitySpline(PdotP_tot, conv, k=1, s=0, ext=1)
+
         probs[i] = prob_dist(Pbdot_meas / Pb)
```

Comment:
Since these are the exact same spline, would it also work to just create `prob_dist` spline first, and then divide `prob_dist(Pbdot_mean / Pb)` by `prob_dist.integral(-np.inf, np.inf)`?

That would avoid a bit of repetition and time, if so.

Reply:
That would indeed ensure the likelihood returned from this function is properly normalized. The reason I did it this way with two splines is so that `conv` by itself is properly normalized as that is the quantity I use when I make the pulsar likelihood plots.

Reply:
Alright, if you think it's more useful like this for whenever we integrate the pulsar plotting I'm fine with it. It's not that slow anyways.